### ISSUE TITLE
Support chunked file uploads

### ISSUE THREAD

**Issue by LucaScorpion** [Tocabi/clearcloud#70]
Currently files can only be uploaded with a single API call, meaning that files that don't fit in either the client or server memory can't be uploaded.

**Comment by ChappIO** [comment]
Have you tested this? I am pretty sure the server supports streamed requests which are dumped to a temporary file.

**Comment by ChappIO** [comment]
It would still be a good feature but I don't think there should be any memory issues with the current implementation.

**Comment by LucaScorpion** [comment]
I did some testing, and it turns out you're absolutely correct. I tried uploading a ~21 gig file (on my machine with 16 gig RAM), and found that it worked fine when running in Docker. Running the server locally did give me an error: `Uncaught error in POST /api/libraries/1/entries: write /tmp/multipart-1223679258: no space left on device`, but that's because my `/tmp` mount is only 8 gig. I'm not sure how that's set up by default on other machines (Windows, Ubuntu, etc) but I don't think it should be an issue there. For now I'll close this issue ^^
